packages/bpk-svgs/gulpfile.babel.js

```diff
@@ -266,4 +268,47 @@ gulp.task('icons-lg', ['icons-common'], () => {
   return merge(react, datauri, rawDatauri);
 });
 
-gulp.task('default', ['elements', 'spinners', 'icons-sm', 'icons-lg']);
+gulp.task('icons-font', ['icons-common'], () => {
+  const generateFont = gulp
+    .src('src/icons/**/*.svg')
+    .pipe(chmod(0o644))
+    .pipe(iconfont({
+      fontName: 'BpkIcon', // required
+      prependUnicode: false,
+      formats: ['ttf', 'eot', 'woff'], // default, 'woff2' and 'svg' are available
+      /**
+       * Normalize and fontHeight(>1000) are needed in order to have all the glyphs rendered
+       * correctly, for more info go to the npm package docs
+       * https://www.npmjs.com/package/gulp-iconfont
+       */
+      normalize: true,
+      fontHeight: 1001,
+    }));
+
+
+  const saveFont = generateFont
+    .pipe(clone())
+    .pipe(gulp.dest('dist/font'));
+
+  const saveMapping = generateFont
+    .on('glyphs', (glyphs) => {
+      // Og all the glyphs generate a key value pair with name and code
+      const mapping = glyphs.reduce((acc, glyph) => {
+        // use punycode to get the text representation of the unicode
+        acc[glyph.name] = punycode.ucs2
+          .decode(glyph.unicode[0])
+          .map(point => `${point.toString(16).toUpperCase()}`).join('');
+        return acc;
+      }, {});
+      // Create a wirable stream to a json file
+      const mappingStream = fs.createWriteStream('dist/iconMapping.json', {
```

Comment:
I think that the mapping should live alongside the fonts in `dist/font/mapping.json`

Reply:
that's the ideal, but when the event is emitted the font folder does not exists, so I either create the folder as part of the task or leave it in the dist folder, what do you think?

Reply:
Unless it will interfere with the rest of the process, I think best to create the folder and have both output files in the same place 😸

Reply:
cool, will do

Reply:
You should be able to pipe the contents of `JSON.stringify(mapping, null, 4)` to a stream which can then be piped to gulp.dest('dist/font')